### PR TITLE
mgr/dashboard_v2: Improve README.md

### PR DESCRIPTION
**Pull request by votdev**
- Add note how to install the npm packages
- Adapt command lines to use local angular-cli installation

Signed-off-by: Volker Theile <vtheile@suse.com>

### CODE REVIEW

**Review by LenzGr — APPROVED** [review]

File: src/pybind/mgr/dashboard_v2/frontend/README.md [hunk 1]
```diff
@@ -1,44 +1,48 @@
...
 This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 1.6.3.
 
+## Installation
+
+Run `npm install` to install the required packages locally.
+
 ## Development server
 
-Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
+Run `npm run start` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
```
Comment (Devp00l) [inline src/pybind/mgr/dashboard_v2/frontend/README.md:11]: You are able to run `ng serve` directly or to run `npm start`. But `npm start` is the same as `npm run start`.  So I would stick to `npm start`. As we might change the `ng serve` command in the future to use a proxy to communicate with our backend elsewhere.

File: src/pybind/mgr/dashboard_v2/frontend/README.md [hunk 2]
```diff
@@ -1,44 +1,48 @@
-# Openattic
+# Dashboard
 
 This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 1.6.3.
 
+## Installation
+
+Run `npm install` to install the required packages locally.
+
```
Comment (Devp00l) [inline src/pybind/mgr/dashboard_v2/frontend/README.md:8]: You should mention here that if someone does not want to install the angular cli globally all commands that begin with `ng` need an additional `npm run` before it.

File: src/pybind/mgr/dashboard_v2/frontend/README.md [hunk 3]
```diff
@@ -1,44 +1,48 @@
...
 ## Development server
 
-Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
+Run `npm run start` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
 
 ## Code scaffolding
 
-Run `ng generate component component-name` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module`.
+Run `npm run ng generate component component-name` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module`.
```
Comment (Devp00l) [inline src/pybind/mgr/dashboard_v2/frontend/README.md:15]: If you add the information mentioned above you don't need to care about the `ng` usage.

File: src/pybind/mgr/dashboard_v2/frontend/README.md [hunk 4]
```diff
@@ -1,44 +1,48 @@
...
 
 ## Build
 
-Run `ng build` to build the project. The build artifacts will be stored in the `dist/` directory. Use the `-prod` flag for a production build.
+Run `npm run build` to build the project. The build artifacts will be stored in the `dist/` directory. Use the `-prod` flag for a production build.
```
Comment (Devp00l) [inline src/pybind/mgr/dashboard_v2/frontend/README.md:19]: You need to mention that `-prod` has to be added to `ng build` or that one has to use `npm run build -- -prod` instead

File: src/pybind/mgr/dashboard_v2/frontend/README.md [hunk 5]
```diff
@@ -1,44 +1,48 @@
...
 
 ## Running unit tests
 
-Run `ng test` to execute the unit tests via [Karma](https://karma-runner.github.io).
+Run `npm run test` to execute the unit tests via [Karma](https://karma-runner.github.io).
```
Comment (Devp00l) [inline src/pybind/mgr/dashboard_v2/frontend/README.md:23]: `npm test` is enough ;)

**Review by ricardoasmarques — APPROVED** [review]

**Pull request merged**